### PR TITLE
Send EnableRbac=true on cluster create, add logs

### PR DESCRIPTION
**Pull request by maxkorp**
https://github.com/rancher/rancher/issues/15230

### CODE REVIEW

File: drivers/aks/aks_driver.go [hunk 1]
```diff
@@ -27,6 +28,9 @@ import (
 	"k8s.io/client-go/rest"
 )
 
+var truePointer = true
+var redactionRegex = regexp.MustCompile("\"(clientId|secret)\": \"(.*)\"")
```
Comment (nathan-jenan-rancher) [inline drivers/aks/aks_driver.go]: can these be made into `const`s?

Comment (maxkorp) [reply drivers/aks/aks_driver.go]: Nein. 
- `truePointer` is used for a pointer assignment, and you can't take the address of a const.
- The regex is returned from a function and can't be made into a const

Comment (nathan-jenan-rancher) [reply drivers/aks/aks_driver.go]: shows what I know, very well

**Comment by nathan-jenan-rancher** [comment]
LGTM

**Comment by cjellick** [comment]
LGTM

**Pull request merged**